modules/ROOT/pages/dw-core-types.adoc

```diff
@@ -88,7 +88,7 @@ and filter it.
 
 Just like the Java class, `Iterator` is designed to be consumed only once. For
 example, if you pass it to a
-https://docs.mulesoft.com/mule-runtime/4.3/logger-component-reference[Logger component],
```

Comment:
TODO: changed to "latest" in latest source

* https://docs.mulesoft.com/mule-runtime/latest/logger-component-reference[Logger component]